goodmap/core.py

```diff
@@ -7,10 +10,35 @@ def does_fulfill_requirement(entry, requirements):
     return all(matches)
 
 
+def sort_by_lat_lon(data: List[Dict[str, Any]], query_params: Dict[str, List[str]]):
+    try:
+        if "lat" in query_params and "lon" in query_params:
+            lat = float(query_params["lat"][0])
+            lon = float(query_params["lon"][0])
+            data.sort(key=lambda x: (x["position"][0] - lat) ** 2 + (x["position"][1] - lon) ** 2)
+            return data
+        return data
+    except (ValueError, KeyError, IndexError):
+        return data
+
+
+def limit(data, query_params, limit: int):
```

Comment:
is `limit` variable used anywhere? it seems it's overriden and original is not used at all.

Reply:
Deleted this variable